python/tools/dht/tests.py

```diff
@@ -371,6 +478,9 @@ def _gottaGetThemAllPokeNodes(self, consumer, hashes, nodes=None):
                 for n in DhtFeatureTest.foreignNodes:
                     nodes.add(n)
 
+    def __normalConditionTest(self):
```

Comment:
@kaldoran, the changes done on `_totallyNormalTest` don't seem to be used, are not complete and don't quite fit into your PR. May be you could revert this, only keep the changes relevant to the PR and make another PR that involves the complete changes around `_totallyNormalTest` that you wanted to make.